### ISSUE TITLE
Span is Above Select Box - Opening The Select Box is Kinda Weird

### ISSUE THREAD

**Issue by peoplecallmefrancois** [adamcoulombe/jquery.customSelect#43]
The customSelect span is above my select box, as a result, I have to click above the actual select box to get this to open, I have tried using CSS to bring the span down but the max I can get it is a few pixels down. Hence, at the moment this is unusable for me.

Do you know how to fix this?

Very much appreciated.

**Comment by todorus** [comment]
My problem may have been the same as yours Francois. I noticed that the position method of jQuery was the culprit, as customSelect uses it to determine where to position the select.

```
//line 53 @ commit 8425a981303845ff595f64218c69120118b1842b
position = $select.position();
```

https://github.com/adamcoulombe/jquery.customSelect/blob/master/jquery.customSelect.js

In the jQuery documentation it says that this is the "current position of an element relative to the offset parent". Apparently the "offset parent" is the first ancestor of the element that has relative positioning. My solution was to wrap the select with a div and set the wrappers position to relative.

I would recommend the admins of this plugin to mention this in their documentation ;)

**Comment by adamcoulombe** [comment]
Should be fixed after reverting to 0.4.1
